src/Microsoft.AspNetCore.Antiforgery/Internal/DefaultAntiforgeryLoggerExtensions.cs

```diff
@@ -0,0 +1,77 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNetCore.Antiforgery.Internal
+{
+    internal static class DefaultAntiforgeryLoggerExtensions
```

Comment:
You can just call this `AntiforgeryLoggerExtensions`. If we add more logging to the package, it will likely go here as well.